pandas/tests/test_groupby.py

```diff
@@ -4594,9 +4594,6 @@ def test_frame_groupby_plot_boxplot(self):
         _skip_if_mpl_not_installed()
 
         import matplotlib.pyplot as plt
-        import matplotlib as mpl
```

Comment:
Currently `Groupby.plot` is tested mostly under `test_graphics.py`, and plotting tests in `test_groupby.py` looks not maintained. Moving remaining to `test_graphics` is better for maintenance.

Separate issue is `test_graphics` gets larger though.